package.json

```diff
@@ -3,106 +3,56 @@
   "version": "1.0.0",
```

Comment:
this changes un-eject the project see [npm run eject](https://create-react-app.dev/docs/available-scripts/#npm-run-eject)